tools/docker/demo/Dockerfile

```diff
@@ -1,40 +1,68 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
-ARG SPARK_VERSION=3.2.0
+ARG MMLSPARK_VERSION=0.9.5
```

Comment:
nit: we might want to rename this to SYNAPSEML_VERSION

Reply:
Fixed